perlbrew-install

```diff
@@ -1,20 +1,29 @@
 #!/bin/sh
 
-if [ "x${PERLBREWURL}" == "x" ]; then
+
+clean_exit () {
+    [ -f $LOCALINSTALLER ] && rm $LOCALINSTALLER
+    exit $1
+}
+
+LOCALINSTALLER=$(mktemp perlbrew.XXXXXX)
+
+if [ -z "${PERLBREWURL}" ]; then
     PERLBREWURL=https://raw.githubusercontent.com/gugod/App-perlbrew/master/perlbrew
 fi
 
 if [ -z "$TMPDIR" ]; then
     if [ -d "/tmp" ]; then
         TMPDIR="/tmp"
+        cd $TMPDIR || clean_exit 1
```

Comment:
Only `cd $TMPDIR` if `$TMPDIR` is not the current directory, but what happens if `$TMPDIR` is set in the environment?  Then no `cd` happens.  Is that correct?